internal/postgres/reconcile.go

```diff
@@ -158,9 +41,31 @@ func WALVolumeMount() corev1.VolumeMount {
 func InstancePod(ctx context.Context,
 	inCluster *v1beta1.PostgresCluster,
 	inInstanceSpec *v1beta1.PostgresInstanceSetSpec,
+	inClusterCertificates, inClientCertificates *corev1.SecretProjection,
 	inDataVolume, inWALVolume *corev1.PersistentVolumeClaim,
 	outInstancePod *corev1.PodSpec,
 ) {
+	certVolumeMount := corev1.VolumeMount{
+		Name:      naming.CertVolume,
+		MountPath: naming.CertMountPath,
+		ReadOnly:  true,
+	}
+	certVolume := corev1.Volume{
+		Name: certVolumeMount.Name,
+		VolumeSource: corev1.VolumeSource{
+			Projected: &corev1.ProjectedVolumeSource{
+				// PostgreSQL expects client certificate keys to not be readable
+				// by any other user.
+				// - https://www.postgresql.org/docs/current/libpq-ssl.html
+				DefaultMode: initialize.Int32(0o600),
```

Comment:
this `o` looks out of place. Should it be this?
```suggestion
				DefaultMode: initialize.Int32(0600),
```

Reply:
`0o` indicates that this is an octal number, like `0x` for hexadecimal. https://golang.org/ref/spec#Integer_literals

I prefer it over just `0` because the latter is too easy to typo. For example, `chmod` accepts a leading `1` to set the sticky bit, but doing that in Go would change to a decimal number.

 ```console
 $ touch cbandy
$ chmod 0600 cbandy
$ ls -ld cbandy
-rw-------. 1 vagrant vagrant 0 Sep 22 21:05 cbandy
$ chmod 1600 cbandy
$ ls -ld cbandy
-rw------T. 1 vagrant vagrant 0 Sep 22 21:05 cbandy
```

https://play.golang.org/p/bEOWEpT0jZ0
```go
fmt.Println(0600)   // 384
fmt.Println(1600)   // 1600
fmt.Println(0o1600) // 896
```

Reply:
Ok cool thanks for the explanation :+1: